libkscreen build is failing with the version bump:

```
$ nix build .#plasma5Packages.libkscreen
CMake Error at /nix/store/77amv8csgx8rbjazd9g6qfk996sqab7z-extra-cmake-modules-5.99.0/share/ECM/modules/ECMPoQmTools.cmake:149 (find_package):
  Could not find a package configuration file provided by "Qt5LinguistTools"
  with any of the following names:

    Qt5LinguistToolsConfig.cmake
    qt5linguisttools-config.cmake

  Add the installation prefix of "Qt5LinguistTools" to CMAKE_PREFIX_PATH or
  set "Qt5LinguistTools_DIR" to a directory containing one of the above
  files.  If "Qt5LinguistTools" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  /nix/store/77amv8csgx8rbjazd9g6qfk996sqab7z-extra-cmake-modules-5.99.0/share/ECM/modules/ECMPoQmTools.cmake:242 (ecm_process_po_files_as_qm)
  CMakeLists.txt:74 (ecm_install_po_files_as_qm)
```

--- a/pkgs/desktops/plasma-5/libkscreen/default.nix
+++ b/pkgs/desktops/plasma-5/libkscreen/default.nix
@@ -10,12 +10,13 @@
 , wayland
 , libXrandr
 , qtx11extras
+, qttools
 }:
 
 mkDerivation {
   pname = "libkscreen";
   nativeBuildInputs = [ extra-cmake-modules wayland-scanner ];
-  buildInputs = [ kconfig kwayland plasma-wayland-protocols wayland libXrandr qtx11extras ];
+  buildInputs = [ kconfig kwayland plasma-wayland-protocols wayland libXrandr qtx11extras qttools ];
   outputs = [ "out" "dev" ];
   patches = [
     ./libkscreen-backends-path.patch

